homeassistant/components/vconnex/const.py

```diff
@@ -0,0 +1,47 @@
+"""Constants for the Vconnex integration."""
+
+from __future__ import annotations
+
+from homeassistant.components.switch import DOMAIN as SWITCH_DOMAIN
```

Comment:
There is a new Platform constant available
```suggestion
from homeassistant.components.const import Platform
```

Reply:
It should be: 
`from homeassistant.const import Platform`